package/python-kiwi-spec-template

```diff
@@ -276,8 +295,12 @@ BuildRequires:  dracut
 %endif
 Requires:       bc
 Requires:       cryptsetup
-%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?fedora} || 0%{?rhel} || 0%{?ubuntu}
+%if 0%{?ubuntu} < 1804
```

Comment:
This will not work correctly, since it'll trigger on Fedora and RHEL/CentOS.

This should be `%if 0%{?ubuntu} && 0%{?ubuntu} < 1804`.